scripts/ci/upload-build

```diff
@@ -2,35 +2,33 @@
 set -ex
 
 # configuration
-BUILD_VARIANT=${BUILD_VARIANT:-''}
-BRANCH_NAME=${BRANCH_NAME:-'master'}
-DIST_PATH=${DIST_PATH:-'dist'}
-PACKAGE_PATH=${PACKAGE_PATH:-'.'}
-SHA_SLUG=${SHA_SLUG:-$(git rev-parse --short HEAD)}
-
-# aws settings
 AWS_BUCKET='downloads.mesosphere.io'
 BUCKET_PATH='dcos-ui'
+BRANCH_NAME=${BRANCH_NAME:-'master'}
+PKG_VERSION=$(git tag -l --contains HEAD)
+
+if [[ $PKG_VERSION ]]
+then
+  PKG_VERSION="-$PKG_VERSION"
+fi
```

Comment:
If this is called from master, `PKG_VERSION` is empty, isnt it? which filename does the `latest` release have? If I get this correct ( https://github.com/dcos/dcos/pull/2506/files ), currently it will lead to a filename without a version or the `latest` suffix 🤔 (`master+dcos-ui.tar.gz`) - we should make sure, that this doesnt happen - but please to not simply add a `else` part here, because we dont want the `latest` release to be default…